apps/prairielearn/elements/pl-overlay/pl-overlay.py

```diff
@@ -110,11 +110,15 @@ def render(element_html: str, data: pl.QuestionData) -> str:
         # so we don't have to worry about all the alignment possibilities
         if left is not None:
             x = left
+        elif right is None or width is None:
+            raise ValueError("Either left or width+right must be specified")
         else:
             x = width - right
```

Comment:
Nit, I'd rather structure this as:

```py
if left is not None:
    # ...
elif right is not None:
    # ...
else:
    raise ValueError(...)
```

I generally prefer to have exception handling come first or last, not be interleaved.

Reply:
Changed in 7a050ae09115469e33cebbb3732cb5a6a4ca89af.